projects/canopy/src/styles/utils.scss

```diff
@@ -3,3 +3,24 @@
 .lg-visually-hidden {
   @include lg-visually-hidden;
 }
+
+$breakpoints: (
+  // 320
+  'sm': 20rem,
+  // 768
+  'md': 48rem,
+  // 1024
+  'lg': 64rem,
+  // 1280
+  'xl': 80rem,
+  // 1440
+  'xxl': 90rem
+) !default;
+
+@each $name, $bp in $breakpoints {
+  .lg-show-at-#{$name} {
```

Comment:
What about the `.lg-hide-at-#{$name}`?

Reply:
Oh I'm a bit confused why this is here and in the `responsive.scss`? is that a mistake or am i missing something?

Reply:
Oh whoops, was meant to remove that, thanks!

Reply:
Removed ✅